controllers/util/k8sutil.go

```diff
@@ -42,6 +42,9 @@ const (
 
 	// This is the name for the OwnerUID FieldIndex
 	OwnerUIDIndexName = "ownerUID"
+
+	// CephBlockPoolBlackListAnnotation is used to disable cephBlockPool mirroring for RDR
+	CephBlockPoolBlackListAnnotation = "ocs.openshift.io/disable-mirroring"
```

Comment:
Possible to use `allow/reject` synonyms?

Reply:
Updated it to use `CephBlockPoolForbidMirroringLabel = "ocs.openshift.io/forbid-mirroring"`